src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -216,6 +225,32 @@ public StateVectorToHtmlResultEncoder(IConfigurationSource configurationSource)
                     vector.SignificantAmplitudes(ConfigurationSource).Select(item =>
                     {
                         var (amplitude, basisLabel) = item;
+                        var displayPhaseAsArrows = ConfigurationSource.PhaseDisplayStyle;
+
+                        //different options for displaying phase style
+                        var phaseCell = ConfigurationSource.PhaseDisplayStyle switch
+                        {
+                            PhaseDisplayStyle.None => FormattableString.Invariant($@"
+                                <td> 
+                                </td>
+                            "),
```

Comment:
Looking at the screenshots you showed me, it may be helpful to not add a cell in this case, rather than an empty cell. That would also take modifying the value of `outputTable` below to not include a `<th>` if `PhaseDisplayStyle` is `None`.
```suggestion
                            PhaseDisplayStyle.None => "",
```